app/code/Magento/Theme/Block/Html/OrderItemsPager.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Theme\Block\Html;
+
+use Magento\Framework\Data\Collection;
+use Magento\Framework\Exception\LocalizedException;
+
+/**
+ * Html pager block
+ *
+ * @SuppressWarnings(PHPMD.ExcessivePublicCount)
+ * @api
+ * @since 100.0.2
+ */
+class OrderItemsPager extends \Magento\Catalog\Block\Product\Widget\Html\Pager
```

Comment:
Could you please clarify why this class was created inside the theme module? This logic is specific and should be inside Sales module.

Reply:
Has been moved to sales module.